frontend/src/components/pages/ComponentLibrary.tsx

```diff
@@ -168,11 +170,27 @@ const ButtonExamples = () => {
   );
 };
 
+const LinkItems: LinkItemProps[] = [
+  {
+    name: "Student Page",
+    urlLink: "/students",
```

Comment:
nit: can we update all the urlLinks to use the exported constants in `src/constants/Routes.ts`
```suggestion
    urlLink: STUDENT_PAGE,
```